docker/Emulator_x86

```diff
@@ -98,7 +98,8 @@ ENV ANDROID_VERSION=$ANDROID_VERSION \
 ENV PATH ${PATH}:${ANDROID_HOME}/build-tools
 RUN echo y | sdkmanager "platforms;android-${API_LEVEL}" && \
     echo y | sdkmanager "system-images;android-${API_LEVEL};${IMG_TYPE};${SYS_IMG}" && \
-    echo y | sdkmanager "emulator"
+    echo y | sdkmanager "emulator" \
```

Comment:
Could you put && before \ like previous line?

Reply:
oh you right my mistake  :s